src/plugins/input_control_vis/public/components/editor/controls_tab.tsx

```diff
@@ -224,8 +220,7 @@ class ControlsTabUi extends PureComponent<ControlsTabUiProps, ControlsTabUiState
   }
 }
 
-export const ControlsTab = injectI18n(ControlsTabUi);
-
-export const getControlsTab = (deps: InputControlVisDependencies) => (
-  props: Omit<ControlsTabUiProps, 'core'>
-) => <ControlsTab {...props} deps={deps} />;
+const ControlsTab = injectI18n(ControlsTabUi);
```

Comment:
nit: what do you think about removing `injectI18n`. We just need to modify a little this file in 4 more places.

Reply:
done 🙂